movies/src/components/Search.jsx

```diff
@@ -0,0 +1,83 @@
+import { useRef, useState } from "react";
+import { makeStyles } from '@mui/styles';
+import constants from "../utils/constants";
+import SearchIcon from '@mui/icons-material/Search';
+
+const useStyle = makeStyles({
+
+    button: {
+        margin: '0',
+        width: '37px',
+        border: 'none',
+        height: '37px',
+        padding: '0',
+        borderRadius: '50%',
+        backgroundColor: '#263238',
+        paddingTop: '3px',
+        cursor: 'pointer',
+        position: 'absolute',
+        zIndex: '5',
+        top: '1px',
+        marginLeft: '1px',
+    },
+
+    "@keyframes InputEffectExit": {
+        "0%": {
+            display: "block !important",
+            width: '0px'
+        },
+        "100%": {
+            width: '218px'
+        },
+    },
+
+    inputEffect: {
+        animation: `$InputEffectExit 1000ms`,
+        animationFillMode: 'forwards',
+    },
+
+}, {
+    name: 'Search'
+})
+
+
+
+
+
+function Search({ setData }) {
+    const [inputValue, setInputValue] = useState()
+    const searchStyle = useStyle()
+    const inputRef = useRef()
+    const [clickCount, setClickCount] = useState(0)
+
+
+    function SearchMovies() {
+        fetch(`${constants.API_PATH}/search/movie?api_key=e366d974f73ae203397850eadc7bce1f&query=${inputValue}`)
```

Comment:
Please use out API ket from coonstants
```suggestion
        fetch(`${constants.API_PATH}/search/movie?api_key=${constants.API_KEY}&query=${inputValue}`)
```

Reply:
Already tried that.
Your code is also copied put does not work.

Reply:
Because our API key is wrong.

In the `constants .s` file our API key is not defined correctly.
it should be `e81b5d12d52c0de91f235d90e155683c`